src/main/java/moze_intel/projecte/utils/TransmutationEMCFormatter.java

```diff
@@ -0,0 +1,22 @@
+package moze_intel.projecte.utils;
+
+public class TransmutationEMCFormatter {
+	public static String EMCFormat(double EMC) {
+		String postFix = "";
+		double magnitude = 1;
+
+		String formats[] = {"Q", "T","B"};
+		Double magnitudes[] = {1e15, 1e12, 1e9};
+
+		for (int i=0; i <formats.length; i++) {
+			double testMag = magnitudes[i];
+			if (EMC >= testMag) {
```

Comment:
I'd do `testMag * 1000` here and also add M to the mix, so it always shows 3 digits. Maybe even `* 1000000`.

```suggestion
			if (EMC >= 1000  * testMag) {
```